src/uct/ib/ud/base/ud_iface.c

```diff
@@ -501,6 +503,10 @@ UCS_CLASS_DEFINE(uct_ud_iface_t, uct_ib_iface_t);
 ucs_config_field_t uct_ud_iface_config_table[] = {
     {"IB_", "", NULL,
      ucs_offsetof(uct_ud_iface_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_ib_iface_config_table)},
+
+    {"RETRY_COUNT", "15", "Transport retries",
```

Comment:
Yes. we should just look at absolute time the remote peer has not responded and not on number of retry attempts, since the interval between retries may not be constant, and don't want to make user aware of the retry algorithm.